frontend/src/Pages/ProviderPlatformManagement.tsx

```diff
@@ -116,6 +140,24 @@ export default function ProviderPlatformManagement() {
                 toaster('Unable to toggle provider state', ToastState.error);
             });
     };
+    const refreshToken = (provider: ProviderPlatform) => {
+        const errorMsg = 'Unable to refresh token for provider for ' + provider.name;
+        API.get<ProviderResponse>(`/provider-platforms/${provider.id}/refresh`)
```

Comment:
the `API` class automatically prepends `/api/` to the request, so this can be 
```ts
API.get<ProviderResponse>(`provider-platforms/${provider.id}/refresh`)
```

Reply:
made change to path to account for /api/